src/GlideConversion.php

```diff
@@ -95,7 +96,15 @@ public function save(string $outputFile)
             return;
         }
 
+        $conversionResultDirectory = pathinfo($this->conversionResult, PATHINFO_DIRNAME);
+
         rename($this->conversionResult, $outputFile);
+
+        $iterator = new FilesystemIterator($conversionResultDirectory);
+
+        if (! $iterator->valid()) {
```

Comment:
The `FilesystemIterator` is used here to check if there are any files left in the directory, not to check whether it's a directory